scenarios/crowdsecurity/aws-bf.md

```diff
@@ -0,0 +1 @@
+# detect aws bruteforce login
```

Comment:
maybe more specific (ie. leakspeed or bucket size) and specify what kind of use-cases it covers (existing user within your org or root account)